Lib/test/test_strptime.py

```diff
@@ -135,6 +135,9 @@ def test_pattern_escaping(self):
                       "%s does not have re characters escaped properly" %
                       pattern_string)
 
+    # bpo-37552 [Windows] strptime/strftime return invalid results with UCRT version 17763.615
+    @unittest.skipIf(sys.platform == 'win32' and locale.getdefaultlocale()[1] == 'cp65001',
```

Comment:
Rather than repeating the whole condition and comment several times, I'd rather see it factored out to a simple `@skip_if_buggy_ucrt` decorator.

Separately, I would suggest that that decorator should actually check for one of the known bugs in the UCRT, and only skip if that bug is still present; we *will* forget to remove these skips later otherwise :)

Reply:
Those are both great ideas :)